README.md

```diff
@@ -45,7 +44,30 @@ const remoteServiceName = 'youtube';
 const zipkinFetch = wrapFetch(fetch, {tracer, remoteServiceName});
 ```
 
-## Instrumentations
+## Browser
+
+The `zipkin` library can be used in the browser. There are a few [examples](https://github.com/openzipkin/zipkin-js-example) available for the `web` and `react-native`.
```

Comment:
should we mention react native?

Reply:
I was assuming the react-native example worked in the browser as well, but I see that it is an Android and iOS example only. I’ll remove this.